tools/Az.Tools.Predictor/Az.Tools.Predictor/AzContext.cs

```diff
@@ -100,13 +101,32 @@ public Version ModuleVersion
             }
         }
 
+        /// <inheritdoc/>
+        public bool IsInternal { get; internal set; }
+
+        /// <summary>
+        /// The survey session id appended to the survey.
+        /// </summary>
+        /// <remarks>
+        /// We only collect this information in the preview. So make it internal and not to make it to the interface.
```

Comment:
My understanding is `surveyId` is appended to URL of survey. I prefer to that you polish this comment such as "will be removed after GA".

Reply:
Will the change look good to you?
```
        /// <remarks>
        /// We only collect this information in the preview and it'll be removed in GA. That's why it's not defined in the
        /// interface IAzContext and it's internal.
        /// </remarks>
```